indico/web/client/js/react/components/principals/util.js

```diff
@@ -126,7 +129,7 @@ export class PrincipalType {
       [PrincipalType.multipassGroup]: Translate.string(
         'This group does not exist anymore. Please choose a different one.'
       ),
-      // event roles are hard-deleted
+      // event roles and category roles are hard-deleted
```

Comment:
absolutely nothing wrong with writing it like this, but personally i'd use the more concise *"event/category roles ..."* here (and in the comment below)